e2e/tests/workspace/workspace-ui.spec.ts

```diff
@@ -18,132 +18,171 @@ describe('Edit Workspace page', () => {
-    expect(await nameInput.isReadOnly()).toBe(false);
-
-    // expect DataSet Select field exists
-    const dataSetSelect = new AouElement(await workspace.select_dataSet());
-    expect(await dataSetSelect.isVisible()).toBe(true);
-  }, 60 * 1000);
-
-  // Click CreateNewWorkspace link in My Workpsaces page => Open Create Workspace page
-  test('My Workspaces page: Click link Create-New-Workspace', async () => {
-    const workspace = new WorkspacePage(page);
-    await workspace.goURL();
-    await workspace.click_button_CreateNewWorkspace();
-    await workspace.waitUntilPageReady();
-  }, 60 * 1000);
-
-  // Checking all fields out-of-box
-  test('Create Workspace page: Question 1', async () => {
-    const workspace = new WorkspacePage(page);
-    await workspace.goURL();
-    await workspace.click_button_CreateNewWorkspace();
-    await workspace.waitUntilPageReady();
-
-    // expand Disease purpose section if needed
-    const expandIcon = await workspace.getResearchPurposeExpandIcon();
-    if (expandIcon !== undefined) {
-      await (expandIcon[0]).click();
+  test('Workspace cards have same UI size', async () => {
+    const workspaceCards = new WorkspaceResourceCard(page);
+    const cards = await workspaceCards.getAllCardsElements();
+    let width;
+    let height;
+    for (const card of cards) {
+      const cardElem = new WebElement(page, card.asElementHandle());
+      expect(await cardElem.isVisible()).toBe(true);
+      const size = await cardElem.size();
+      if (width === undefined) {
+        width = size.width; // Initialize width and height with first card element's size, compare with rest cards
+        height = size.height;
+      } else {
+        expect(size.height).toEqual(height);
+        expect(size.width).toEqual(width);
+      }
     }
-    // Disease-focused research checkbox
-    const diseaseName = workspace.diseaseName();
-    let cbox = (await diseaseName.checkbox());
-    expect(await cbox.isVisible()).toBe(true);
-    expect(await cbox.getProp('checked')).toBe(false);
-    expect(await cbox.getProp('disabled')).toBe(false);
-    const txtField = await diseaseName.textfield();
-    expect(await txtField.isVisible()).toBe(true);
-    expect(await txtField.getProp('disabled')).toBe(true);
-
-    // Set the checkbox checked
-    await page.evaluate(elem => elem.click(), await (await diseaseName.checkbox()).asElement() );
-    // TODO wait async for checked and disabled checking or test will fail
+  });
+
+  // Click CreateNewWorkspace link on My Workpsaces page => Open Create Workspace page
+  test('Click Create New Workspace link on My Workspaces page', async () => {
+    const workspaces = new WorkspacesPage(page);
+    await workspaces.goToURL();
+    const workspaceEdit = await workspaces.clickCreateNewWorkspace();
+    const workspaceNameTextbox = await workspaceEdit.getWorkspaceNameTextbox();
+    expect(await workspaceNameTextbox.isVisible()).toBe(true);
+  });
+
+
+/*
```

Comment:
commented out broken tests for now. needs fixing.

Reply:
Please fix or remove this code before merging

Reply:
created https://precisionmedicineinitiative.atlassian.net/browse/RW-4531 to keep track of code that need to fix. commented out code removed here.